app/assets/stylesheets/application.scss

```diff
@@ -759,16 +766,39 @@ span.inactive {
 
   text-align: center;
   text-overflow: ellipsis;
+
+  &.card-chip-green {
+    background: rgba(78, 206, 0, 0.15);
+    color: rgba(44, 115, 0, 1);
+  }
+
+  &.card-chip-amber {
+    background: rgba(252, 247, 193, 0.8);
```

Comment:
It'd be nice if we can use variables for the colours. It'll become easier to refactor later if the definitions are in one place.